### ISSUE TITLE
Active Development Branch

### ISSUE THREAD

**Issue by maebert** [kvh/parsedatetime#1]
Hi there, just wanted to know whether you'll accept pull requests here or on the google gode repo?

**Comment by kvh** [comment]
The google code repo is the official one as far as I know. (I am not the author/maintainer.) That said, the code hasn't really been touched over there since 2009. I would definitely accept any patches on this repo :)

**Comment by maebert** [comment]
Ah, I see. The thing is, I'm writing on something that depends on parsedatetime (http://github.com/maebert/jrnl), so I would like to see it not only patched in a dev repo, but upstream so that users can obtain an up-to-date version from the cheeseshop. But maybe we could get things rolling again if we start here?

**Comment by kvh** [comment]
yeah, let me reach out to the original developer and get his thoughts.

On Thu, May 24, 2012 at 11:48 AM, Manuel Ebert <
reply@reply.github.com

**Comment by bear** [comment]
I migrated the code to github a couple months ago as I find myself spending more and more time in github now.

Please do either patch it here or submit pull requests over on the official repo:

http://github.com/bear/parsedatetime
